### ISSUE TITLE
redesign: GetNotificationText is throwing NRE's

### ISSUE THREAD

**Issue by Trolldemorted** [signal-csharp/Signal-Windows#73]
Incoming sync messages are treated as outgoing, and thus their author is null:

```
AdaptiveText title = new AdaptiveText()
{
    Text = message.Author.ThreadDisplayName,
    HintMaxLines = 1
};
```

Should we suppress notifications for incoming sync messages? Signal-Android is removing its notifications and marking the conversation as read when a sync message arrives.

**Comment by golf1052** [comment]
I meant to only have notifications be sent on incoming messages but I forgot to put the line in. Fixed with e9350f7658c1d7f7d7ad60284df351676e199f3e

**Comment by Trolldemorted** [comment]
Should we clear the notification for a thread if a message which is not incoming appears?

I think we definitely should remove the notification, not sure what should be done with the tile.
